drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -262,6 +267,28 @@ local function level_attr_handler(driver, device, ib, response)
   end
 end
 
+local function level_min_attr_handler(driver, device, ib, response)
+  if ib.data.value ~= nil then
+    local level = math.floor((ib.data.value / 254.0 * 100) + 0.5)
```

Comment:
Could be helpful to add a description for the operations done to obtain `level`

Reply:
Added a comment in 2cd1a7ca5dbafa8046c75650aa9a59365a24c2ce.